apps/api/src/cart/cart.controller.ts

```diff
@@ -30,6 +31,13 @@ export class CartController {
     return cart;
   }
 
+  @Get('/:id')
+  async getCart(@Param('id') userID: string) {
+    const userCart = await this.cartService.getCart(userID);
+    if (!userCart) throw new NotFoundException('Cart is empty');
+    return userCart;
+  }
+
```

Comment:
utilize our access token to identify user. no need to pass id as param.

Reply:
@brentmillen resolved sir please check recent changes. thanks